radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java

```diff
@@ -1,22 +1,29 @@
 package com.radixdlt.client.atommodel.timestamp;
 
-import com.radixdlt.client.atommodel.quarks.ChronoQuark;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.radixdlt.client.core.atoms.particles.Particle;
+import java.util.HashMap;
+import java.util.Map;
+import org.radix.serialization2.DsonOutput;
 import org.radix.serialization2.SerializerId2;
 
 /**
  * Particle which stores time related aspects of an atom.
  */
 @SerializerId2("TIMESTAMPPARTICLE")
 public class TimestampParticle extends Particle {
+	@JsonProperty("timestamps")
+	@DsonOutput(DsonOutput.Output.ALL)
+	private Map<String, Long> timestamps = new HashMap<>();
+
 	private TimestampParticle() {
 	}
 
 	public TimestampParticle(long timestamp) {
-		super(new ChronoQuark("default", timestamp));
+		this.timestamps.put("default", timestamp);
 	}
 
 	public long getTimestamp() {
-		return getQuarkOrError(ChronoQuark.class).getTimestamp();
+		return timestamps.get("default");
```

Comment:
Will this change the API (what type of error did `getQuarkOrError` throw)?

Reply:
Or maybe it doesn't matter...

Reply:
There was really no way for the `TimestampParticle` to be constructed without a `ChronoQuark`, so the "OrError" was not really an actual thing that happened.  The intention was more to signal a programming logic failure of some kind.

Reply:
👍